README.md

```diff
@@ -537,14 +573,26 @@ The `@include` directive can be combined with the `__PATH__` [variable](#builder
 @include __PATH__ + "/../shared/Constants.shared.nut"
 ```
 
-#### GitHub Authentication ####
+#### Authentication ####
+
+##### GitHub #####
 
 When using GitHub `@include` statements, authentication is optional. However, you should bear in mind that:
 
 - If you use authentication, the GitHub API provides much higher rate limits.
 - Authentication is required to access private repositories.
 
-Apart from a GitHub username, you need to provide either a [personal access token](https://github.com/settings/tokens) **or** a password (which is less secure and not recommended). If you are using Builder as a [library](#library-installation), GitHub credentials can be stored using your system's environment variables or in files that store Builder variables. When you are using Builder's [command line tool](#command-line-tool-installation), your GitHub credentials will need to be passed into the `pleasebuild` command.
+Apart from a GitHub username, you need to provide either a [personal access token](https://github.com/settings/tokens) **or** a password (which is less secure and not recommended).
+
+##### Bitbucket Server #####
```

Comment:
The minimum version of Bitbucket Server that is required to support this feature should be listed here.

Reply:
You're right, I will add it